### ISSUE TITLE
Undefined identifiers after C/C++ plugin update

### ISSUE THREAD

**Issue by ltcperera** [Microsoft/vscode#27006]
- VSCode Version: Code 1.12.2 (19222cdc84ce72202478ba1cec5cb557b71163de, 2017-05-10T13:16:25.899Z)
- OS Version: Linux x64 4.4.0-78-generic
- Extensions: 

|Extension|Author|Version|
|---|---|---|
|code-gnu-global|austin|0.2.2|
|cpptools|ms-vscode|0.11.1|
|clang-format|xaver|1.4.0|;

Open a C/C++ source file and notice that identifiers that are defined in C/C++ headers are being flagged as not found (seeing an underline under each symbol). Selecting "Problems" from the view menu displays these identifiers as undefined. 

This occurred only after I updated the C/C++ plugin. Never had this issue before that.

I used the snippet below to repro, but can repro this with multiple C/C++ files. 
---

Steps to Reproduce:

1. Enter the code below and note that "cout" and "endl" are flagged as undefined symbols. 

#include <iostream>

using namespace std;

int main()
{
  if (true)
  {
    // Testing
  }
  cout << "Testing" << endl;
  return 0;
}

**Comment by rongjiecomputer** [comment]
You should report bugs about vscode-cpptools at https://github.com/Microsoft/vscode-cpptools

Please read https://github.com/Microsoft/vscode-cpptools/blob/master/Documentation/LanguageServer/FAQ.md

**Comment by ThatJCB** [comment]
I have the same but only after updating to VSCode 1.12.2
The upgrade to 1.12.2 triggers the extension I have installed to re-install. As far as I can remember the version numbers of the extensions are unchanged. In my case this is the C/C++ and the Arduino extension.
Headers in libraries from both are treated incorrectly.

I can work round the issue by changing the setting in C/C++ for Intellisense from 'Default' to 'Tag Parser'. The 'Tag Parser' appears to correctly recognise the libraries.

I thought that Intellisense was part of VSCode, not part of the Extension, so I think there is an argument for this issue to be here. Should we report in it C/C++ and Arduino? 
Neither of those appear to have changed recently!

I leave it to someone who knows how the code is structured to decide the best place to report this.

There are a couple of screenshots at the very end of one of my blog posts:
[http://blog.discoverthat.co.uk/2017/04/installing-visual-studio-code-for.html](http://blog.discoverthat.co.uk/2017/04/installing-visual-studio-code-for.html)

**Comment by ThatJCB** [comment]
This has also been reported in the C/C++ extension issues.
[https://github.com/Microsoft/vscode-cpptools/issues/743](https://github.com/Microsoft/vscode-cpptools/issues/743)
